### ISSUE TITLE
chore: add accessibility features where needed

### ISSUE THREAD

**Issue by RitaDee** [open-sauced/app#2019]
I'll be utilizing regEx to identify areas in need of accessibility improvements. These [regEx patterns]((<img\b(?![^>]*alt=)[^>]*>|<input\b(?![^>]*<label\b)[^>]*>|<a\b(?![^>]*>[^<]*</a>)[^>]*>|<[^>]*(?!(aria-)[^>]*>))) will help me locate specific elements within the codebase that require accessibility enhancements.

**Comment by RitaDee** [comment]
I will .take this.

**Comment by nickytonline** [comment]
Before beginning this @RitaDee, can we discuss the issue? What is the regex checking for exactly?

Also, there's [tools like Axe](https://www.deque.com/axe/) to help uncover accessibility issues.

**Comment by RitaDee** [comment]
> Before beginning this @RitaDee, can we discuss the issue? What is the regex checking for exactly?
> 
> Also, there's [tools like Axe](https://www.deque.com/axe/) to help uncover accessibility issues.

The combined regular expression is used to identify potential accessibility issues within HTML code. It's to catch specific patterns of 
code that may indicate missing or problematic accessibility features, like missing "alt" attributes for images, missing labels for form inputs, empty anchor tags, or elements without ARIA attributes. 

I can explore the stated tool too, if you permit me to

**Comment by RitaDee** [comment]
> > Before beginning this @RitaDee, can we discuss the issue? What is the regex checking for exactly?
> > Also, there's [tools like Axe](https://www.deque.com/axe/) to help uncover accessibility issues.
> 
> The combined regular expression is used to identify potential accessibility issues within HTML code. It's to catch specific patterns of code that may indicate missing or problematic accessibility features, like missing "alt" attributes for images, missing labels for form inputs, empty anchor tags, or elements without ARIA attributes.
> 
> I can explore the stated [tools like Axe](https://www.deque.com/axe/) too (it doesn’t look like it is a free extension), if you permit me to

<img width="1440" alt="image" src="https://github.com/open-sauced/app/assets/106407013/3f97b556-681f-4d61-8ac7-3a65ae3a5dcd">

**Comment by nickytonline** [comment]
It's a free browser extension. There's a paid product as well, but it's not required. Feel free to use the regex as well, but open separate issues for the things to fix.

Thanks for looking into this!

**Comment by RitaDee** [comment]
> It's a free browser extension. There's a paid product as well, but it's not required. Feel free to use the regex as well, but open separate issues for the things to fix.
> 
> Thanks for looking into this!

@nickytonline, I need clarity on how to tackle this based on what you stated "open separate issues for the things to fix."  So, you want me to fix them in bits right? Take a look at this:
<img width="1438" alt="Screenshot 2023-10-31 at 22 34 18" src="https://github.com/open-sauced/app/assets/106407013/f59b0e96-71f3-426c-bb5a-1a1672855dd6">

**Comment by nickytonline** [comment]
Yeah fix them in bits, but also don't tackle them right away as we do have some internal initiatives to work on this as well, so it's better to open the issues, have a discussion and then move forward.

Already I see on that page the links which are the crayon and right chevron icons, do not have text to say what they are for.

![CleanShot 2023-10-31 at 17 41 49](https://github.com/open-sauced/app/assets/833231/64668987-e305-4d47-a970-c96110e2857d)

**Comment by RitaDee** [comment]
> Yeah fix them in bits, but also don't tackle them right away as we do have some internal initiatives to work on this as well, so it's better to open the issues, have a discussion and then move forward.

Okay. Thanks

**Comment by RitaDee** [comment]
> > Yeah fix them in bits, but also don't tackle them right away as we do have some internal initiatives to work on this as well, so it's better to open the issues, have a discussion and then move forward.
> 
> Okay. Thanks

What is your thought on this:
https://github.com/open-sauced/app/issues/2028?

**Comment by nickytonline** [comment]
@RitaDee, I'm going to go ahead and close this as you're opening up separate issues for things a11y issues you discover. Thanks for working on all this!

**Comment by RitaDee** [comment]
> @RitaDee, I'm going to go ahead and close this as you're opening up separate issues for things a11y issues you discover. Thanks for working on all this!

Sure @nickytonline!
